docs/cst.md

```diff
@@ -0,0 +1,223 @@
+ value-chains or microfinance assessments, local language translation, user reviews, open-source programming, etc.) can be
+ crowdsourced and are often paid via mobile-phone-based micropayments.
+
+### Step 3: What Types of Information and Tools Do I need?
+ Crowdsourcing can deliver information on any topic as long as the
+ relevant information is directly accessible to the participating
+ individuals. This information is mostly provided as text, numbers, or
+ Interacive Voice Response (IVR), but can
+ sometimes contain photos, video imagery, or geo-tagged data, if
+ offered by the chosen crowdsourcing software or platform. For market
+ data observation or polling (opinions, votes, ideas), specific tools
+ are available ([Links to Further Sources](#links-to-further-sources)).
+
+### Step 4: How to Make Collaborating with the Crowd Possible?
+
+ The effective application of crowdsourcing requires indepth local
+ knowledge and clearly identified information needs to engage and
+ empower the relevant crowd. As a minimum requirement, the crowd needs
+ to have access to the required information and needs to be able to
+ submit information as simply as possible. Crowds can be reached through open calls for participation via
+radio, TV, or online or direct engagement in bounded crowdsourcing. Further considerations when engaging crowds are:
+
+**Motivation** 
+
+-   Vision (clearly defined ideals and objectives, e.g., government accountability)
+
+-   Material incentives (e.g., micro-payments through mobile tools for
+    accomplished tasks)
+
+**Skillsets**
+
+-   Communication skills (e.g., local languages, digital literacy,
+    \[functional\] literacy)
+
+-   Technical and professional skills (e.g., type of profession)
+
+**IT infrastructure**
+
+-   Ownership of crowdsourcing devices (e.g., simple vs. smart phone
+    users; ITU for country data (see [Links to Further Sources](#links-to-further-sources))
+
+-   Access to relevant infrastructure (e.g., cost for and connectivity
+    to the internet)
+
+### Step 5: Selecting Crowdsourcing Tools
+
+ The selection of a crowdsourcing tool depends on both the information need and the characteristics of the crowd. For example, women sometimes do not possess their own phone, but share it. In general,
+ user interfaces need to be as simple and engaging as possible to
+ motivate participation.
+
+-   **Mobile-based systems:** Open-source, mobile-based systems provide
+    customizable crowdsourcing solutions for any information need that
+    allows for data collection via SMS, social media, and direct upload
+    of data by the crowd. **SMS**-based systems offer the crowd a
+    two-way communication to select relevant information or optional
+    answers similar to surveys. Similarly, **Unstructured Supplementary Service Data (SSD)** solutions allow the crowd to select options in pre-designed protocols and the project to receive brief survey data. **IVR** allows the crowd to
```

Comment:
is the formatting only in the code or also in the actual sheet displayed like this? Not coherent compared to the above text